README.md

```diff
@@ -52,26 +52,26 @@ gulp.task('templates', function() {
 
 All options supported by the [Jade API](http://jade-lang.com/api/) are supported
```

Comment:
The docs are still at http://jade-lang.com/ according to https://github.com/pugjs/pug do you want it just changed to [Pug API](http://jade-lang.com/api/)